### PR TITLE
IT BUILDS

### PR DESCRIPTION
**Pull request by katherinebernstein**

### CODE REVIEW

**Comment by makosblade** [comment]
I approve of this message

File: src/main/java/com/openlattice/datastore/data/controllers/DataController.java [hunk 1]
```diff
@@ -1117,6 +1119,10 @@ private WriteEvent clearOrDeleteEntities( UUID entitySetId, Set<UUID> entityKeyI
         return new WriteEvent( maxVersion, numUpdates );
     }
 
+    private void performAssociationAccessChecksForDelete( UUID entitySetId, DeleteType deleteType ) {
+        // TODO
```
Comment (makosblade) [inline src/main/java/com/openlattice/datastore/data/controllers/DataController.java]: 🤔

**Review by makosblade — APPROVED** [review]

File: src/main/java/com/openlattice/datastore/data/controllers/DataController.java (hunk 1 above)
Comment (you-leee) [inline src/main/java/com/openlattice/datastore/data/controllers/DataController.java]: Just curious: what access checks are missing here? We do some checks in `deleteAssociations`

Comment (katherinebernstein) [reply src/main/java/com/openlattice/datastore/data/controllers/DataController.java]: oh whoops, didn't realize that was a local change. removing...

**Review by you-leee — APPROVED** [review]

File: src/main/java/com/openlattice/datastore/data/controllers/DataController.java [hunk 2]
```diff
@@ -1086,6 +1086,8 @@ private WriteEvent clearOrDeleteEntities( UUID entitySetId, Set<UUID> entityKeyI
         final Map<UUID, PropertyType> authorizedPropertyTypes =
                 getAuthorizedPropertyTypesForDelete( entitySetId, Optional.empty(), deleteType );
 
+
+
```
Comment (makosblade) [inline src/main/java/com/openlattice/datastore/data/controllers/DataController.java]: 🤔

**Pull request merged**